src/runtime_src/core/include/xrt/xrt_bo.h

```diff
@@ -44,7 +44,7 @@ typedef uint32_t xrtMemoryGroup;
  * @details
  * Use when constructing xrt::bo from xclBufferHandle
  */
-struct xcl_buffer_handle { xclBufferHandle bhdl; };
+struct xcl_buffer_handle { xrt_buffer_handle bhdl; };
```

Comment:
Oops, we can't do this.  This is a binary incompatibility in an end user exposed API.
Granted the API that uses this type is undocumented and explicitly states that it's a bad idea to mix xcl objects with xrt objects, but still we need to think how to make this not break the ABI.

Reply:
Updated the PR with the fix we had discussed. Please take a look.